prediction_prophet/app.py

```diff
@@ -148,18 +148,18 @@ def research(
             with st.status("Making prediction"):
                 prediction = _make_prediction(market_question=question, additional_information=report, agent=Agent("gpt-4o", model_settings=ModelSettings(temperature=0.0)))
 
-                if prediction.outcome_prediction == None:
+                if prediction is None:
```

Comment:
`ProbabilisticAnswer` from `_make_prediction` can not be None, so this can be removed now.

Reply:
removed